src/components/admin/AppLayout.vue

```diff
@@ -1,19 +1,24 @@
 <template>
```

Comment:
Click on menu item should close dropdown

![image](https://user-images.githubusercontent.com/5394573/63718564-87575880-c853-11e9-9069-db89a25e4bd4.png)